app/src/test/scala/org/alephium/explorer/persistence/queries/UnconfirmedTransactionQueriesSpec.scala

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018 The Alephium Authors
+// This file is part of the alephium project.
+//
+// The library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the library. If not, see <http://www.gnu.org/licenses/>.
+
+package org.alephium.explorer.persistence.queries
+
+import scala.util.Random
+
+import org.scalacheck.Gen
+import slick.jdbc.PostgresProfile.api._
+
+import org.alephium.explorer.AlephiumFutureSpec
+import org.alephium.explorer.Generators._
+import org.alephium.explorer.persistence.{DatabaseFixtureForEach, DBRunner}
+import org.alephium.explorer.persistence.model._
+import org.alephium.explorer.persistence.schema._
+import org.alephium.util.TimeStamp
+
+class UnconfirmedTransactionQueriesSpec
+    extends AlephiumFutureSpec
+    with DatabaseFixtureForEach
```

Comment:
I think you could use `DatabaseFixtureForAll` here, since you are anyway deleting the data every time